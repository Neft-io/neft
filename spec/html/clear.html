<b id="2"><!-- abc --> a b c

</b>