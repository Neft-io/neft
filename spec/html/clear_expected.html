<b id="2">a b c</b>